src/app/effects/blockly-editor.effects.ts

```diff
@@ -119,43 +116,42 @@ export class BlocklyEditorEffects {
-                  Blockly.Blocks[name] = block;
+                    Blockly.Blocks[name] = block;
                 }
-                const LeaphyTheme = Blockly.Theme.defineTheme('leaphy', {
+                config.theme = Blockly.Theme.defineTheme('leaphy', {
                     'blockStyles': defaultBlockStyles,
                     'categoryStyles': categoryStyles,
                     'componentStyles': componentStyles,
                     name: 'leaphy',
-                  })
-                  config.theme = LeaphyTheme;
-                  const parser = new DOMParser();
-                  const toolboxXmlDoc = parser.parseFromString(baseToolboxXml, 'text/xml');
-                  const toolboxElement = toolboxXmlDoc.getElementById('easyBloqsToolbox');
-                  const leaphyCategories = parser.parseFromString(leaphyToolboxXml, 'text/xml');
-                  const leaphyRobotCategory = leaphyCategories.getElementById(robotType.id);
-                  if (robotType.showLeaphyActuators) {
-                      const leaphyExtraCategory = leaphyCategories.getElementById(`${robotType.id}_actuators`);
-                      toolboxElement.prepend(leaphyExtraCategory);
-                  }
-                  toolboxElement.prepend(leaphyRobotCategory);
-                  const serializer = new XMLSerializer();
-                  const toolboxXmlString = serializer.serializeToString(toolboxXmlDoc);
-                  config.toolbox = toolboxXmlString;
-                  // @ts-ignore
-                  const workspace = Blockly.inject(element, config);
-                  const toolbox = workspace.getToolbox();
-                  toolbox.getFlyout().autoClose = false;
-                  const xml = Blockly.utils.xml.textToDom(startWorkspaceXml);
-                  Blockly.Xml.domToWorkspace(xml, workspace);
-                  this.blocklyState.setWorkspace(workspace);
-                  this.blocklyState.setToolboxXml(toolboxXmlString);
-                  if (this.blocklyState.workspaceStatus == WorkspaceStatus.Clean) {
-                    this.backEndWiredEffects.send('restore-workspace-temp', robotType.id);
-                  }
-                  toolbox.selectItemByPosition(0);
-                  toolbox.refreshTheme();
+                });
+                const parser = new DOMParser();
+                const toolboxXmlDoc = parser.parseFromString(baseToolboxXml, 'text/xml');
+                const toolboxElement = toolboxXmlDoc.getElementById('easyBloqsToolbox');
+                const leaphyCategories = parser.parseFromString(leaphyToolboxXml, 'text/xml');
+                const leaphyRobotCategory = leaphyCategories.getElementById(robotType.id);
+                toolboxElement.prepend(leaphyRobotCategory);
+                if (robotType.showLeaphyActuators) {
+                    const leaphyExtraCategory = leaphyCategories.getElementById(`${robotType.id}_extra`);
```

Comment:
_extra is _actuators geworden. En toolboxElement.prepend ipv. append. Kortom: de code die er stond was oke.